x-pack/filebeat/input/httpjson/config_auth.go

```diff
@@ -82,6 +82,8 @@ type oAuth2Config struct {
 	// common oauth fields
 	ClientID       string              `config:"client.id"`
 	ClientSecret   string              `config:"client.secret"`
+	User           string              `config:"user"`
```

Comment:
Could you please put these properties in alpha-num order?

Reply:
Sure. I will update it.